devices/wholeBodyDynamics/KnownExternalWrench.cpp

```diff
@@ -0,0 +1,127 @@
+#include "KnownExternalWrench.h"
+
+namespace wholeBodyDynamics
+{
+
+KnownExternalWrench::KnownExternalWrench::KnownExternalWrench()
+{
+}
+
+KnownExternalWrench::KnownExternalWrench(std::string parsedframeName, std::string parsedType, std::vector<double> parsedDirection, std::vector<double> parsedPosition)
+{
+    frameName = parsedframeName;
+    type = parsedType;
+    position = parsedPosition;
+    direction = parsedDirection;
+    
+    if(type=="full")
```

Comment:
you could add an enum class instead of comparing strings.

Reply:
Why didn't we choose to use the same enum types as defined in iDynTree? FULL_WRENCH, PURE_FORCE, PURE_FORCE_WITH_KNOWN_DIRECTION ?

Reply:
> Why didn't we choose to use the same enum types as defined in iDynTree? FULL_WRENCH, PURE_FORCE, PURE_FORCE_WITH_KNOWN_DIRECTION ?

addressed in https://github.com/robotology/whole-body-estimators/pull/48/commits/edcd509dad868124ec98e627ecd260af425c8ca1


> you could add an enum class instead of comparing strings which might be prone to error.

I am not sure it's worth creating an enum and a map variables just for these 3 words, what do you think?